### ISSUE TITLE
Test release process

### ISSUE THREAD

**Issue by loredanacirstea** [raiden-network/raiden-contracts#446]
Test release process from https://github.com/raiden-network/raiden-contracts/blob/master/RELEASE.rst and remove the comment when done:

> This document has not been tested by someone else than @loredanacirstea.

If anything is wrong or incomplete, a PR should be opened.

- [x] bump contracts version
- [x] deploy contracts on testnets
- [x] verify contracts with Etherscan
- [x] bump package version
- [x] make a PyPI release

**Comment by pirapira** [comment]
@loredanacirstea when I'm deploying the services contracts to Ropsten, what shall I use for `TOKEN_USED_TO_PAY_SERVICES`?

```
python -m raiden_contracts.deploy services --rpc-provider http://127.0.0.1:8545 --private-key /path/to/your/private_key/file --gas-price 10 --gas-limit 6000000 --token-address TOKEN_USED_TO_PAY_SERVICES
```

**Comment by loredanacirstea** [comment]
@palango - any preference?
We can use `WETH`, considering our token registration limits. Or the `CustomToken` that we have in our tests.

**Comment by palango** [comment]
Will the new deployment have limits and be limited to `WETH` as well? Then `WETH` makes most sense, otherwise it might be nice to have a `mint`able token.
@karlb Opinions on your side?

Different question: When do you plan to do the deployment? Can we maybe get in the MS and UDC changes?

**Comment by pirapira** [comment]
> Will the new deployment have limits and be limited to WETH as well?

Yes.  So I go with `WETH`.

> When do you plan to do the deployment?

This week or the next week latest.

> Can we maybe get in the MS and UDC changes?

I don't mind repeating the deploy process.

**Comment by loredanacirstea** [comment]
I already added the MSC and RSBC to the deployment process in https://github.com/raiden-network/raiden-contracts/pull/449. 
Once the UDC PR is merged, @karlb can add it.

**Comment by pirapira** [comment]
> I already added the MSC and RSBC

That sounds like an ABI change. Should the next version number be 0.5.0. instead of 0.4.1?

**Comment by loredanacirstea** [comment]
Yes.

**Comment by karlb** [comment]
> Will the new deployment have limits and be limited to `WETH` as well? Then `WETH` makes most sense, otherwise it might be nice to have a `mint`able token.
> @karlb Opinions on your side?

No preferences, WETH seems fine.

**Comment by pirapira** [comment]
Etherscan verification is failing for me on all three test networks.  I'm seeing
```
{'status': '0', 'message': 'NOTOK', 'result': 'Fail - Unable to verify'}
```

**Comment by pirapira** [comment]
but I was able to register the source code manually on Etherscan.

**Comment by loredanacirstea** [comment]
Yes, it is a known weird issue: https://github.com/raiden-network/raiden-contracts/issues/349.

**Comment by pirapira** [comment]
I made #459 after bumping the package version.

**Comment by pirapira** [comment]
This has been done.
